scripts/bw2h5.py

```diff
@@ -70,151 +36,113 @@
 _logger.addHandler(_handler)
 
 
-def parse_args():
-    """Parse command line arguments.
-
-    Return:
-        args : parsed argument object.
+def bw2h5(noisybw, cleanbw, layersbw, cleanpeakbw, batch_size, nolabel,
+          nonzero, intervals_file, out_dir, prefix, pad):
+    """Convert bigwig files to h5.
+
+    Args:
+        noisybw: BigWig file containing noisy data.
+        cleanbw: BigWig file containing clean data.
+        layersbw: BigWig file containing layers data.
+        cleanpeakbw: BigWig file containing clean peaks data to be used as
+        labels for training.
+        batch_size: Number of lines to read at a time, since all of the data
+        does not fit in memory.
+        nolabel: No peak labels to encode into h5. Used for generating
```

Comment:
Do we need this option? Can we assume that if label files are not provided, we should encode the noisy data alone?